tests/unit/test_base_command.py

```diff
@@ -17,8 +17,10 @@
 
 @pytest.fixture
 def fixed_time() -> Iterator[None]:
-    with patch("time.time", lambda: 1547704837.040001 + time.timezone):
-        yield
+    year2019 = 1547704837.040001 + time.timezone
+    with patch("time.time", lambda: year2019):
+        with patch("time.time_ns", lambda: year2019 * 1e9):
```

Comment:
Not particularly important but `time.time_ns` returns an int but this function now returns a float, I think it would make sense for consistency to do something like `int(year2019 * 1e9)`